contrib/experimental/great_expectations_experimental/expectations/expect_multicolumn_sum_values_to_be_equal_to_single_column.py

```diff
@@ -85,12 +96,14 @@ class ExpectMulticolumnSumValuesToBeEqualToSingleColumn(MulticolumnMapExpectatio
     examples = [
         {
             "data": {
-                "col_a": [3, 6, 0, 1],
-                "col_b": [-6, -3, 1, 2],
-                "col_c": [1, 0, -1, 3],
-                "col_d": [-2, 3, 0, 6],
-                "col_e": [3, 6, 0, 1],
-                "col_f": [-3, 3, 1, 3],
```

Comment:
What do you think about including a few test cases with integer values as well as floats to make sure the existing functionality still works?

Reply:
Sure!